integrations/content.yml

```diff
@@ -27,3 +29,9 @@ features:
   - Vue components in Markdown
   - Full-text search
   - Table of contents generation
+articles:
+  - name: Create a Blog with Nuxt Content
+    link: https://nuxtjs.org/blog/creating-blog-with-nuxt-content
+    author: Debbie O'Brien
```

Comment:
Shall we rename to authors and each being an object?

Reply:
not sure I understand. can you show me an example?

Reply:
https://github.com/nuxt/integrations/pull/10/commits/e1de6c91ad9a82aa9d2b81ea12dc98dd0a853bf7